language/move-vm/runtime/src/runtime.rs

```diff
@@ -201,17 +201,17 @@ impl<'alloc> VMRuntime<'alloc> {
     }
 }
 
-/// Verify if the transaction arguments match the type signature of the main function.
-fn verify_actuals(signature: &FunctionSignature, args: &[Value]) -> bool {
-    if signature.arg_types.len() != args.len() {
+/// Verify if the transaction actuals match the type signature of the main function.
+fn verify_actuals(parameters: &Signature, args: &[Value]) -> bool {
```

Comment:
Nit: `fn verify_actuals` => `fn verify_args`?

Reply:
to your point, actuals, parameters, arguments (and abbreviation) are very inconsistent right now.
I think renaming is something that should be done careful and also let people in other components to make decision on their code. 
And the inconsistency is that when you put it all together (runtime, verifier, test generation, prover, tools) it touches a lot.
So in this diff you'll find renaming half done and crazy stuff like that but I don't think any alternative is any better